src/containers/Accounts/AccountTransactionTable/AccountTransactionTable.tsx

```diff
@@ -10,12 +10,14 @@ import { ANALYTIC_TYPES, analytics } from '../../shared/utils'
 
 export interface AccountTransactionsTableProps {
   accountId: string
-  currencySelected: string
+  currencySelected?: string
```

Comment:
```suggestion
  currencySelected: string
```
The prop has a default value, so it's not optional

Reply:
I believe it errored without it.